sys/include/net/gnrc/netdev2.h

```diff
@@ -102,6 +103,16 @@ typedef struct gnrc_netdev2 {
      * @brief general information for the MAC protocol
      */
     uint16_t mac_info;
+
+    /**
+     * @brief device's l2 address
+     */
+    uint8_t  l2_addr[IEEE802154_LONG_ADDRESS_LEN];
```

Comment:
> So, do you think I should close this PR? ;-) (simply continue to use gnrc_netdev2->dev->driver->get())

Maybe a good idea to store the address regardless, just to save some time and code.

> BTW, is there a way/clue to get/reach netdev2_ieee802154_t.long_addr and netdev2_ieee802154_t.short_addr ?

Use `dev->driver->get()` with [`NETOPT_ADDRESS_LONG`](http://doc.riot-os.org/group__net__netopt.html#gga19e30424c1ab107c9c84dc0cb29d9906a3f2a397a9a0dbf9673b3501566b05093) or [`NETOPT_ADDRESS`](http://doc.riot-os.org/group__net__netopt.html#gga19e30424c1ab107c9c84dc0cb29d9906a211ad84f5ea3987ed591deeaf1fd4758). respectively